style.css

```diff
@@ -0,0 +1,920 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
+  height: 4rem;
+  background-color: #3e3c3c;
+  display: flex;
+  align-items: center;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+}
+
+.nav-logo {
+  color: #fff;
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+}
+
+.logo-text {
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 20px;
+  color: #fff;
+  font-family: "Inter", sans-serif;
+}
+
+.nav {
+  color: #fff;
+  text-decoration: none;
+  width: 100%;
+  margin: 0 8rem;
+  display: flex;
+  justify-content: space-between;
+}
+
+.ham {
+  display: flex;
+  justify-content: space-between;
+  visibility: hidden;
+}
+
+.nav-link-container {
+  display: flex;
+  align-items: center;
+  font-family: "Inter", sans-serif;
+  font-size: 15px;
+  font-weight: 600;
+  line-height: 20px;
+}
+
+@media screen and (max-width: 768px) {
```

Comment:
- [ ] kindly consider enclosing all the desktop version CSS in this`@media screen and (min-width: 768px) {}` media query as you did here on the mobile version  to make your code more organized and make sure that this requirement is full filled 
> Use media queries in your CSS file to make your desktop version look like the template you selected `(Desktop: >=768px).`